app/build.gradle

```
@@ -13,20 +18,57 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
     }
     buildTypes {
+        debug {
+            // FIXME debug/release에 따라서 base url, key 등이 따로 쓰이도록 설정해주세요. 개발환경과 배포환경에서 쓰이는 값들이 분리되어야 합니다.
```

`p3;` 일단 `FIXME`의 경우 버그인데 현재 상황에서 고치지 못하는 경우 다는 주석입니다. `TODO`로 바꾸는 것이 좋을 것 같습니다.
```suggestion
            // TODO@xxx debug/release에 따라서 base url, key 등이 따로 쓰이도록 설정해주세요. 개발환경과 배포환경에서 쓰이는 값들이 분리되어야 합니다.
```